tests/unit/test_desy.py

```diff
@@ -121,6 +121,7 @@ def test_faulty_marc():
     path = os.path.abspath('tests/unit/responses/desy/faulty_record.xml')
     with open(path, 'r') as xmlfile:
         data = xmlfile.read()
-    result = spider._hep_records_from_marcxml([data])
-    assert result[0]['error'] == "ValueError(u'Unknown string format',)"
-    assert result[0].get('traceback') is not None
+    result = spider._parsed_items_from_marcxml([data])
+    assert result[0].exception == "ValueError(u'Unknown string format',)"
```

Comment:
Sometimes you use `["key"]`, and sometimes you use `.key`, I'd standardize to the latter (as it's more generic in the sense that if we ever change the object to inherit from something else than dict, it will still work, though I don't really thing it will be an issue...).